spec/cfonb/operation_spec.rb

```diff
@@ -81,14 +81,15 @@
     end
 
     context 'with a MMO detail' do
-      let(:detail) { OpenStruct.new(body: '', detail_code: 'MMO', detail: 'USD200000000001234') }
+      let(:detail) { OpenStruct.new(body: '', detail_code: 'MMO', detail: 'USD000000000008358300000001077') }
```

Comment:
real value

Reply:
the previous one was also real 🤔

Reply:
can you please keep the previous one as it tests the `scale`

Reply:
but this one with exchange_rate :P

Reply:
can you please keep the previous one as it tests the scale

Reply:
done. actually, there was a bug. thanks @frantisekrokusek